### ISSUE TITLE
GCop179 false alarm

### ISSUE THREAD

**Issue by sungam3r** [Geeksltd/GCop#263]
```csharp
 internal class Formatter
    {
        public static void Format(string name)
        {
            Console.WriteLine($"{name,10}"); // OK
        }
    }
```

```csharp
 internal class Formatter
    {
        public static void Format(string name)
        {
            Console.WriteLine($"{name,11}"); // GCop179	Do not hardcode numbers, strings or other values. Use constant fields, enums, config files or database as appropriate.	
        }
    }
```

It seems to me that the rule should not be applied to formatting strings.

**Comment by Foroughi** [comment]
@sungam3r The reason why GCop is not showing any warning for the first sample is b/c some of the constant values which are listed below have been excluded intentionally. But I managed the rule not kick in for interpolated string texts. 

 "-2", "-1", "0", "1", "2", "3", "4", "5", "12", "1024", "true", "false", "10", "100", "0.1", "0.5", "0.01", "0.001", "0.0001", "0.00001", "1000", "360"

**Comment by sungam3r** [comment]
Another case that surprised me (net46):
```c#
#pragma warning disable 618 // GCop179	Do not hardcode numbers, strings or other values. Use constant fields, enums, config files or database as appropriate.
```

**Comment by sungam3r** [comment]
Also this:

```c#
#pragma warning disable 618 // GCop179	Do not hardcode numbers, strings or other values. Use constant fields, enums, config files or database as appropriate.
```

**Comment by marjanjavid** [comment]
@Foroughi This rule still have the previous problem on GCop2.6.1-rc2

**Comment by Karvan** [comment]
@azadimehdi we need to stop kicking GCop179 when we see it's commented out or start with # like this example.

`#pragma warning disable 618 // GCop179	Do not hardcode numbers, strings or other values. Use constant fields, enums, config files or database as appropriate.`

**Comment by azadimehdi** [comment]
@Karvan  This issue has already been solved.

**Comment by sungam3r** [comment]
In what release version?

**Comment by Karvan** [comment]
@sungam3r It should be resolved in the latest version which is 2.6.2-rc1

**Comment by sungam3r** [comment]
Nope. This issue still exists.

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.2</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="GCop.All.Geeks" Version="2.6.2-rc.1" />
  </ItemGroup>
</Project>
```

```c#
using System;

namespace ConsoleApp10
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
        }
    }

    internal class Formatter
    {
        public static void Format(string name)
        {
            Console.WriteLine($"{name,11}"); // Warning	GCop179	Do not hardcode numbers, strings or other values. Use constant fields, enums, config files or database as appropriate.	
        }
    }
}
```

**Comment by Karvan** [comment]
@azadimehdi could you please have another look into that.

@sungam3r thank you for updating this one. could you please let me know which version of GCop.Common have you installed on your project? if that's 2.6.2-rc.1 then we definitely should be able to replicate that.

**Comment by sungam3r** [comment]
https://www.nuget.org/packages/GCop.All.Geeks/2.6.2-rc.1
![изображение](https://user-images.githubusercontent.com/21261007/71679086-cc5a4100-2d97-11ea-97e5-c02ea8ae2da0.png)

**Comment by sungam3r** [comment]
https://www.nuget.org/packages/GCop.All.Common/2.6.2-rc.1
![изображение](https://user-images.githubusercontent.com/21261007/71679188-0a576500-2d98-11ea-92ad-45a25c358ec3.png)
